### ISSUE TITLE
Dialog jank during signin animation on Chrome only

### ISSUE THREAD

**Issue by jaredhirsch** [mozilla/persona#3668]
When the chevron ( div.arrow or div.arrowContainer ) animates rightward after a successful login, you can see a flashing dark vertical line move across the dialog, too.

I can't repro on Safari or FF, but it's bad on Chrome/Mac. I'm not having luck getting a screen capture of the animation in progress.

**Comment by edwindotcom** [comment]
i'm see a hard line at all times on FF beta with Retina 
![screen shot 2013-07-18 at 9 46 37 am](https://f.cloud.github.com/assets/3698618/820661/b8e3adfe-efc9-11e2-9bd0-0224ac153bf6.png)

**Comment by vladikoff** [comment]
http://screencast.com/t/QBjoWJ64Tam uploaded a video demo, you can see the line show up right in the end

**Comment by mattbasta** [comment]
+1 on this. I see this on every browser that I use. Seems like a rounding error; increasing the value by half a pixel should probably fix it.

**Comment by jaredhirsch** [comment]
@mattbasta This animation is based on [CSS transitions](https://github.com/mozilla/browserid/blob/train-2013.07.03/resources/static/dialog/js/misc/helpers.js#L23-L40), I think we've got a funky webkit bug here.

Not sure what you mean about a rounding error--this vertical line isn't visible (for me) except during the slide animation. Care to elaborate?

**Comment by jaredhirsch** [comment]
@mattbasta actually, I think I get what you were saying. If I reposition the arrow 1 px to the left, I can't repro the line. Probably a subpixel rendering error because there's no overlap between the elements--I guess this is what you were talking about. Thanks for the hint :beers:

**Comment by mattbasta** [comment]
> If I reposition the arrow 1 px to the left, I can't repro the line. 

Yep, that's it. Thanks for this, I tried digging for the CSS and fixing it myself but mucking with production stylesheets made me lose interest :)

**Comment by bnolan** [comment]
Im still getting this in safari. 
![screen shot 2015-12-17 at 11 56 29 am](https://cloud.githubusercontent.com/assets/17499/11856421/744891fc-a4b5-11e5-8280-4c098e2ac284.png)

**Comment by shane-tomlinson** [comment]
This is still seen, re-opening.
